java/ql/lib/change-notes/2023-04-19-deprecated-execcallable.md

```diff
@@ -0,0 +1,4 @@
+---
+category: deprecated
+---
+* The `ExecCallable` class in `ExternalProcess.qll` has been deprecated.
```

Comment:
Should `ProcessBuilderConstructor`, `MethodProcessBuilderCommand`, and `MethodRuntimeExec` also be explicitly mentioned in the change note? Or is the mention of the `ExecCallable` deprecation enough since they all extend `ExecCallable`? (Again asking for my own understanding).

Reply:
I thought about this, but I think it's implied...? I don't believe we have strict guidelines about this, so it seems like personal preference at this point. If you or someone else (@coadaflorin?) have a strong opinion, happy to change it, but IMHO it's fine as it is.

Reply:
I don't have a strong opinion 🙂 so it's fine to me if left as is.